### DISCUSSION TITLE
E-Mail template variables

### DISCUSSION THREAD

**Discussion by hknobbe** [docusealco/docuseal#160] [Q&A]
Hi!

Is there any documentation about the variable fields to be used in the mail templates? There are a few examples, however I'm wondering if there are more? 

Thanks!

**Comment by omohokcoj** [comment]
@hknobbe it's not in the docs yet but you can find those variable names here:
https://github.com/docusealco/docuseal/blob/master/lib/replace_email_variables.rb#L4

**Reply by omohokcoj** [reply]
@hknobbe you can also find email template variables here: https://www.docuseal.co/guides/send-documents-for-signature-via-api
